### PR TITLE
[12.0][MIG] product_packaging_type & product_packaging_type_required

### PR DESCRIPTION
**Pull request by AdriaGForgeFlow**
This is a standard back-port PR of the modules **product_packaging_type** and **product_packaging_type_required** from v13.

Superseeding #525 

cc ~ @ForgeFlow @guewen @jgrandguillaume

### CODE REVIEW

File: product_packaging_type/README.rst [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+To create a new packaging type:
+
+#. Go to *Sales > Configuration > Products > Product Packaging Type*.
```
Comment (AaronHForgeFlow) [inline product_packaging_type/README.rst]: I can't see this menu. Apart of product packaging do I need to check any other setting  in settings?

Comment (AaronHForgeFlow) [reply product_packaging_type/README.rst]: Got it. It is in Inventory menu.

**Review by AaronHForgeFlow — APPROVED** [review]
Code review + functional tests. LGTM :+1:

File: product_packaging_type/README.rst (hunk 1 above)
Comment (AdriaGForgeFlow) [reply product_packaging_type/README.rst]: Fixed! :+1: Good catch

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM

**Comment by emagdalenaC2i** [comment]
@AdriaGForgeFlow 
I think you should create 2 separated PR one for each module

**Comment by AdriaGForgeFlow** [comment]
> @AdriaGForgeFlow
> I think you should create 2 separated PR one for each module

I thought so at the beginning but I saw that the 13.0 (#536) and the superseeded 12.0 (#525) PR included both of them so I followed the same approach

**Review by MiquelRForgeFlow — APPROVED** [review]

**Review by jarroyomorales — APPROVED** [review]
Code review LGTM! :+1:  No major changes

**Review by etobella — APPROVED** [review]
LGTM :+1:

**Review by JordiBForgeFlow — APPROVED** [review]

**Pull request merged**